packages/catalog-process/src/services/catalogService.ts

```diff
@@ -808,7 +808,7 @@ export function catalogServiceBuilder(
       if (
         descriptor.docs.some(
           (d) =>
-            d.id !== apiEServiceDescriptorDocumentUpdateSeed.id &&
+            d.id !== documentId &&
```

Comment:
Hi @taglioni-r in this [PR](https://github.com/pagopa/interop-be-monorepo/pull/720/files) you added this check but in the model it is not present. Could you please check if `documentId` is correct in place of what you did?

Reply:
Yes exactly it was just an impossible case 👍